db/seeds/run-seed.js

```diff
@@ -1,9 +1,17 @@
-const devData = require('../data/development-data/index.js');
-const seed = require('./seed.js');
-const db = require('../connection.js');
+const devData = require("../data/development-data/index.js");
+const testData = require("../data/test-data/index.js");
+const seed = require("./seed.js");
+const db = require("../connection.js");
+
+const ENV = process.env.NODE_ENV || "development";
+
+const data = {
+  test: testData,
+  development: devData,
+};
 
 const runSeed = () => {
-  return seed(devData).then(() => db.end());
+  return seed(data[ENV]).then(() => db.end());
 };
```

Comment:
I'd rather see the test data seeded directly here. Running an additional ENV check in the testing suite is repeating work unnecessarily. We don't want the tests to ever be run with the dev data.